packages/playground/src/app.jsx

```diff
@@ -17,7 +17,7 @@ const liveSettingsSchema = {
   type: 'object',
   properties: {
     liveValidate: { type: 'boolean', title: 'Live validation' },
-    disable: { type: 'boolean', title: 'Disable whole form' },
+    disabled: { type: 'boolean', title: 'Disable whole form' },
```

Comment:
This file was recently converted to typescript and these contents moved into a different file